inc/saiswitch.h

```diff
@@ -2756,6 +2756,23 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_NAT_EVENT_NOTIFY,
 
+    /**
+     * @brief Number of ECMP members supported by switch
```

Comment:
Please clarify that this is:

      Number of ECMP members supported by switch across Nexthop Group

Reply:
https://github.com/opencomputeproject/SAI/commit/409d507e41bfdada9adbacee112475684ebf249c